modules/router/templates/router_include.conf.erb

```diff
@@ -63,6 +63,8 @@ location ~ ^/contact/govuk/(|service-feedback|problem_reports|foi|page_improveme
 location / {
   <%= scope.function_template(['router/fragments/_basic_auth.erb']) -%>
 
+  add_header Link "<https://assets.<%= @app_domain %>>; rel=preconnect";
```

Comment:
I wonder if this makes more sense to go in the meta tags in the HTML of pages because it might get more visibility there, people might not think to look here for that?